cve_bin_tool/cvedb.py

```diff
@@ -552,6 +554,56 @@ def populate_affected(self, affected_data, cursor, data_source):
         except Exception as e:
             LOGGER.info(f"Unable to insert data for {data_source} - {e}")
 
+    def metric_finder(self, cursor, cve):
+        # SQL query to retrieve the metrics_name based on the metrics_id
```

Comment:
This comment is helpful; maybe we could turn it into a docstring so it can be parsed and used as such?

```suggestion
        ''' SQL query to retrieve the metrics_name based on the metrics_id '''
```

Short explainer on docstrings in case you're not familiar with them: https://www.programiz.com/python-programming/docstrings

More generally: cvedb is lacking in a lot of docstrings but it would be nice to have them in new functions if you remember.   I'll flag a few more new functions in this PR for you.  Sorry I didn't think to do it with earlier PRs!